### PR TITLE
Main Budget Page + fixes

### PR DESCRIPTION
**Pull request by bycho91**
- Created UI for Main Budget Page
- Fixed some stuff for carousel of budget cards according to Jonathan's comments
- User now has access to a button that'll allow them to go back to the specific budgets main page
- Navigation bar no longer has 'My Budget' option. Only Home and Logout

### CODE REVIEW

File: client/src/components/Navigation/Navigation.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ import logo from "../../assets/budgety_small_logo.png";
 import { GiHamburgerMenu } from "react-icons/gi";
 import { ImCancelCircle } from "react-icons/im";
 import { AiOutlineHome } from "react-icons/ai";
-import { MdOutlineEventNote } from "react-icons/md";
+// import { MdOutlineEventNote } from "react-icons/md";
```
Comment (Jmartimus) [inline client/src/components/Navigation/Navigation.tsx:17]: Do you need this?

File: client/src/components/Navigation/Navigation.tsx [hunk 2]
```diff
@@ -49,14 +49,14 @@ const Navigation: React.FC<Props> = ({ user, setUser }) => {
           <>
             <SidebarWrapper>
               <NavIcon to="/">
-                <h2>Dashboard</h2>
+                <h2>Home</h2>
               </NavIcon>
             </SidebarWrapper>
-            <SidebarWrapper>
+            {/* <SidebarWrapper>
```
Comment (Jmartimus) [inline client/src/components/Navigation/Navigation.tsx:55]: Or this part?

File: client/src/components/Navigation/Navigation.tsx [hunk 3]
```diff
@@ -106,15 +106,15 @@ const Navigation: React.FC<Props> = ({ user, setUser }) => {
             <SidebarWrapper>
               <NavIcon to="/" onClick={() => setSidebar(false)}>
                 <AiOutlineHome size="2rem" />
-                <h2>Dashboard</h2>
+                <h2>Home</h2>
               </NavIcon>
             </SidebarWrapper>
-            <SidebarWrapper>
+            {/* <SidebarWrapper>
```
Comment (Jmartimus) [inline client/src/components/Navigation/Navigation.tsx:112]: Or this one?

**Review by Jmartimus — CHANGES_REQUESTED** [review]
Just a few stray things.

File: client/src/components/Navigation/Navigation.tsx (hunk 2 above)
Comment (bycho91) [reply client/src/components/Navigation/Navigation.tsx]: deleted

File: client/src/components/Navigation/Navigation.tsx (hunk 3 above)
Comment (bycho91) [reply client/src/components/Navigation/Navigation.tsx]: deleted

File: client/src/components/Navigation/Navigation.tsx (hunk 1 above)
Comment (bycho91) [reply client/src/components/Navigation/Navigation.tsx]: deleted

**Review by Jmartimus — APPROVED** [review]
Nice!

**Pull request merged**